ValyriaTear.nsi

```diff
@@ -72,6 +72,8 @@ Section "MainSection" SEC01
   File /oname=valyriatear.mo po\es.mo
   SetOutPath "$INSTDIR\po\gl\LC_MESSAGES"
   File /oname=valyriatear.mo po\gl.mo
+  SetOutPath "$INSTDIR\po\gd\LC_MESSAGES"
```

Comment:
Also wrong alphabetical order (but that's the case for `en_GB` too, so likely not an issue.